api/types.proto

```diff
@@ -802,3 +811,17 @@ message RemovedNode {
 	// was issued to this node.
 	Timestamp expiry = 2;
 }
+
+message EncryptedRecord {
+	enum Algorithm {
+		option (gogoproto.goproto_enum_prefix) = false;
+		option (gogoproto.enum_customname) = "Algorithm";
+
+		NONE = 0 [(gogoproto.enumvalue_customname) = "NotEncrypted"];
```

Comment:
The raft data itself will always be encrypted with a DEK, but the DEK itself may not be encrypted, and this type is also used to wrap the encrypted DEK.